jenkins/end_to_end_tests.sh

```diff
@@ -17,7 +17,14 @@ virtualenv venv
 
 pip install -r requirements/base.txt
 
-# Install pages.
+# Install the page objects from the edx-platform repo.
+# Before doing so, we don't need optimizations for lxml,
+# so install it this way which doesn't bother compiling them.
+STATIC_DEPS=true CFLAGS="-O0"  pip install "lxml==3.4.4"
 paver install_pages
+
+# Set the display to the virtual frame buffer (Xvfb)
+export DISPLAY=:1
```

Comment:
I wonder if we want to do anything here to improve bok-choy logging so it's clearer when this error occurs. I'm not sure if we can get enough information from selenium, but thought I'd ask.